.github/workflows/firebase-prod-merge.yml

```diff
@@ -32,7 +33,10 @@ jobs:
           cp .npmrc functions/.npmrc
           echo "//npm.pkg.github.com/:_authToken=${{ secrets.GH_TOKEN_DISCO_MIR }}" >> functions/.npmrc
```

Comment:
should this be the generic github token?

Reply:
oh, can it be? I thought default was only for current repo

Reply:
even though we don't need the token for package installs anymore? I guess we don't need any of this .npmrc stuff?  Also, if we do, I think we should make this a CJ token instead of tied to you.

Reply:
The plan was to remove it.. I just wasn't sure if the repo was ready. I was nervous to make CJ tokens for GitHub as a security practice... but we could have a conversation about that. That's why I added the MIR, so at least we know who it came from. But I agree, it's very inconvenient. I'm just not sure what should be our best practice here.... we should chat. This is also relevant for our collaborator orgs, where CJ is not a member